tests/providers/amazon/aws/operators/test_ecs.py

```diff
@@ -314,3 +314,22 @@ def test_reattach_successful(self, launch_type, tags, start_mock, check_mock, wa
         self.assertEqual(
             self.ecs.arn, 'arn:aws:ecs:us-east-1:012345678910:task/d8c67b3c-ac87-4ffe-a847-4785bc3a8b55'
         )
+
+    @mock.patch.object(
+        ECSOperator, '_cloudwatch_log_events', return_value=[{"message": "Log output", "timestamp": 1000}]
+    )
+    def test_execute_xcom_with_log(self, _):
+        self.ecs.do_xcom_push = True
+        self.assertEqual(self.ecs.execute(None), "Log output")
```

Comment:
```suggestion
    def test_execute_xcom_with_log(self, mock_cloudwatch_log_events):
        self.ecs.do_xcom_push = True
        log_message = self.ecs.execute(context=None)
        assert log_message == mock_cloudwatch_log_events.return_value[0]['message']
```
WDYT? :)

Reply:
You want to make sure that the _real output_ (`log_message`) is equal to the one of the mocked one.

Reply:
> LGTM, just a minor "improvement". WDYT? :)

Yeah, definitely an improvement - thanks!

I avoided the `log_message` assignment since it's only used once.